### PR TITLE
Zinkerz/v6

### PR DESCRIPTION
**Pull request by OfirZinkerz**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "Zinkerz"
   ],
   "dependencies": {
-    "infra": "https://github.com/Knowledge-OTP/infra.git#~v5.0.80",
+    "infra": "https://github.com/Knowledge-OTP/infra.git#zinkerz/v6",
```
Comment (assafshp) [inline bower.json]: next release

File: bower.json [hunk 2]
```diff
@@ -27,9 +27,5 @@
     "mockfirebase": "~0.11.0",
     "jasmine-jquery": "^2.1.1",
     "aws-sdk": "aws-sdk-js#^2.7.14"
```
Comment (assafshp) [inline bower.json]: add resoultions

File: test/spec/liveSession/liveSessionSrv.spec.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
+// describe('testing service "LiveSessionSrv":', function () {
+//     'use strict';
+//
+//     beforeEach(module('znk.infra-web-app.liveSession', 'htmlTemplates', 'storage.mock', 'testUtility', 'user.mock', 'env.mock'));
+//
+//     beforeEach(module('znk.infra-web-app.liveSession', function($provide, PresenceServiceProvider, LiveSessionSubjectSrvProvider, znkAnalyticsSrvProvider, CallsUiSrvProvider) {
+//         localStorage.setItem('email', 'ofir+actEdu@zinkerz.com');
+//         localStorage.setItem('password', '123123');
```
Comment (assafshp) [inline test/spec/liveSession/liveSessionSrv.spec.js]: try to work

**Comment by OfirZinkerz** [comment]
jenkins test

**Comment by OfirZinkerz** [comment]
jenkins-test

**Comment by assafshp** [comment]
please fix

**Comment by OfirZinkerz** [comment]
jenkins-test

**Pull request merged**